crates/solidity/inputs/language/src/definition.rs

```diff
@@ -5687,6 +5776,452 @@ codegen_language_macros::compile!(Language(
                                 reserved = Till("0.7.1"),
                                 value = Atom("years")
                             )]
+                        ),
+                        // Yul EVM Builtins
```

Comment:
I suggest keeping the list of keywords in this topic alphabetically sorted, and not divide them based on usage (built-in or not). That division is defined by the `Enum` item above, and the two lists are prone to going out of sync over time. It is also easier to find/look-up keywords this way.

We already have IDE support for find references/go to definition for the DSL (and also simple grep) if that is needed.

```suggestion
```